dist/zoomist.css

```diff
@@ -1 +1 @@
r .zoomist-slider-wrapper{background-color:var(--zoomist-slider-track-color-hover)}.zoomist-slider:hover .zoomist-slider-bar{background-color:var(--zoomist-slider-bar-color-hover)}.zoomist-slider:hover .zoomist-slider-button:before{box-shadow:0 0 8px #0009}.zoomist-slider-horizontal .zoomist-slider-wrapper{align-items:center;justify-content:flex-start;width:var(--zoomist-slider-bar-size);height:var(--zoomist-slider-bar-side)}.zoomist-slider-horizontal .zoomist-slider-bar{top:0;width:calc(var(--value) * 1%);height:100%}.zoomist-slider-horizontal .zoomist-slider-button{left:calc(var(--value) * 1%)}.zoomist-slider-vertical .zoomist-slider-wrapper{align-items:flex-end;justify-content:center;width:var(--zoomist-slider-bar-side);height:var(--zoomist-slider-bar-size)}.zoomist-slider-vertical .zoomist-slider-bar{bottom:0;width:100%;height:calc(var(--value) * 1%)}.zoomist-slider-vertical .zoomist-slider-button{bottom:calc(var(--value) * 1%)}.zoomist-slider-wrapper{position:relative;display:flex;background-color:var(--zoomist-slider-track-color);transition:background-color var(--zoomist-modules-tranistion-duration)}.zoomist-slider-bar{position:absolute;z-index:0;left:0;display:block;border-radius:var(--zoomist-slider-bar-border-radius);background-color:var(--zoomist-slider-bar-color);transition:background-color var(--zoomist-modules-tranistion-duration)}.zoomist-slider-button{position:relative;z-index:1;display:block;width:0!important;height:0!important}.zoomist-slider-button:before{content:"";position:absolute;left:calc(var(--zoomist-slider-button-size) * -.5);top:calc(var(--zoomist-slider-button-size) * -.5);width:var(--zoomist-slider-button-size);height:var(--zoomist-slider-button-size);display:block;background-color:var(--zoomist-slider-button-color, #fff);border-radius:50%;box-shadow:0 0 4px #0006;transition:box-shadow var(--zoomist-modules-tranistion-duration)}.zoomist-zoomer{position:absolute;z-index:2;top:0;right:0;border-radius:0 0 0 4px;overflow:hidden}.zoomist-zoomer-button{position:relative;display:flex;justify-content:center;align-items:center;width:var(--zoomist-zoomer-button-size);height:var(--zoomist-zoomer-button-size);background:none;background-color:var(--zoomist-zoomer-button-color);border:0;transition:background-color var(--zoomist-modules-tranistion-duration);cursor:pointer}.zoomist-zoomer-button:hover{background-color:var(--zoomist-zoomer-button-color-hover)}.zoomist-zoomer-button:hover .zoomist-zoomer-icon{fill:var(--zoomist-zoomer-icon-color-hover)}.zoomist-zoomer-button.zoomist-zoomer-disabled{pointer-events:none;background-color:var(--zoomist-zoomer-button-color-disabled);opacity:var(--zoomist-zoomer-button-opacity-disabled)}.zoomist-zoomer-button.zoomist-zoomer-disabled .zoomist-zoomer-icon{fill:var(--zoomist-zoomer-icon-color-disabled)}.zoomist-zoomer-icon{width:var(--zoomist-zoomer-icon-size);height:var(--zoomist-zoomer-icon-size);fill:var(--zoomist-zoomer-icon-color);transition:fill var(--zoomist-modules-tranistion-duration)}
```

Comment:
any ideas why do we added prefixed `-webkit-user-select:none` when we didn't change CSS files?

Reply:
Rebuilt without changes in css now 🤔 Not sure what was the reason before